packages/core/test/acceptance/styling_spec.ts

```diff
@@ -801,6 +802,124 @@ describe('styling', () => {
     });
   });
 
+  onlyInIvy('ViewEngine only supports !important when set through the `style` attribute')
+      .it('should set !important on a single property', () => {
+        // We go through `CSSStyleDeclaration.setProperty` to set `!important`. Values set through
+        // `setProperty` aren't propagated to the `style` attribute on IE, even though the style is
```

Comment:
I'd propose to include IE version here (and in other cases below) - it'd be easier to find it later (once IE 11 support is deprecated and removed in the future).
```suggestion
        // `setProperty` aren't propagated to the `style` attribute on IE 11, even though the style is
```